### DISCUSSION TITLE
👋 Welcome to TensorLy Discussions!

### DISCUSSION THREAD

**Discussion by JeanKossaifi** [tensorly/tensorly#218] [General]
👋 Welcome to TensorLy Discussions!

------------------------------

We are using discussions as a place for the TensorLy community to discuss, exchange ideas and connect.  Here you can:

* 🙋‍♂️🙋‍♀️🙋 Ask any questions you may have
* 🚨 Check about a new feature before you open a pull-request
* 💡 Suggest and discuss new ideas
* 🪲 Report bugs
* 📽 Discuss the API and more generally the project
* 🤝 Connect and exchange with other members of the TensorLy community

**Don't be shy, introduce yourself and join the discussion!**

_Github discussion is currently in Beta so we may expect some bugs so bear with us if this happens!_

**Comment by cohenjer** [comment]
Hi Jean, hi all

This looks great, looking forward to discussing with everyone 🎉 

Working with my collegue Caglayan Tuna on the NNCP class right now, things are going smoothly hopefully we can start the PR soon! 🙏 

Also thanks for adding tl.clip() in the backend, really helpful as a more versatile replacement for np.maximum(). 👍

**Comment by QhL-UNC** [comment]
Hi Jean, hi all,

Thank you very much for creating Tensorly. 
I currently work on tensor regression for one project using longitudinal data. Some of the predictors are n obs * number of features * number of days (time series features), and some of the predictors are n obs * number of features (demographics). The response is a scalar response. 
I first changed the time series features to tensor format (as predictors) using tl.tensor() function, and used the scalar response, and I built TuckerRegression and CPRegression, respectively. There were no errors or warnings when I built these models, but the predicted Y values are all "nan". 
Did I miss something when I built the model using TuckerRegressor or CPRegressor functions? 
I look forward to hearing from you.

Sincerely,
Qinghua

**Reply by JeanKossaifi** [reply]
It's hard to answer without having more information. Can you open a separate discussion for this? 
Can you check whether your input has any nan values? 
Thanks!

**Reply by QhL-UNC** [reply]
Sorry for the late reply. Thank you for replying quickly for my questions. I will open a new discussion.
